wolfcrypt/src/port/Renesas/renesas_tsip_util.c

```diff
@@ -72,10 +174,12 @@ int tsip_hw_lock()
 {
     int ret = 0;
 
-    WOLFSSL_MSG("enter esp_sha_hw_lock");
+    /*WOLFSSL_MSG("enter esp_sha_hw_lock");*/
```

Comment:
Looks like we need to remove this debug message line?

Reply:
Originally, those message were active. However ,they are very noisy when debug mode is activated, since they are used in primitive locking functions. I think they should be removed.